core/unit_test/TestMathematicalFunctions.hpp

```diff
@@ -1581,6 +1585,7 @@ struct TestIsFinite {
       ++e;
       Kokkos::printf("failed isfinite(float)\n");
     }
+#if !(defined(KOKKOS_ENABLE_CUDA) && defined(KOKKOS_COMPILER_MSVC))
```

Comment:
What is the deal with commenting out some of the half type tests?.  Neither the commit messages nor the description refer to these changes.

Reply:
ah yeah I guess missed that in the message: some random error inside system header files ...